src/Table2Beta/SelectedRowsHeader.tsx

```diff
@@ -0,0 +1,104 @@
+import * as React from "react";
+
+import Cell from "./Cell";
+import "./SelectedRowsHeader.less";
+import { FlexBox } from "../../src";
+
+interface Props {
+  selectedRows: Set<any>;
+  contentType?: { singular: string; plural?: string };
+  actions: Array<ActionInput>;
+  allSelected: boolean;
+}
+
+export interface ActionInput {
+  callback(selectedRows: Set<any>): void;
+  title: { singular: string; plural?: string };
+  // should icon be required?
+  icon?: string;
+}
+
+const cssClasses = {
+  ROW: "Table2Beta--selectedRowsHeader",
+  TITLE_CELL: "Table2Beta--selectedRowsHeader--titleCell",
+  ACTIONS_CELL: "Table2Beta--selectedRowsHeader--actionsCell",
+  ACTIONS_FLEXBOX: "Table2Beta--selectedRowsHeader--actionsFlexbox",
+  ACTION: "Table2Beta--selectedRowsHeader--action",
+  ACTION_ICON: "Table2Beta--selectedRowsHeader--actionIcon",
+  ACTION_TITLE: "Table2Beta--selectedRowsHeader--actionTitle",
+};
+
+export default function SelectedRowsHeader({
+  selectedRows,
+  contentType,
+  actions,
+  allSelected,
+}: Props) {
+  const rowsAreSelected = selectedRows.size > 0;
+  const singleRowSelected = selectedRows.size === 1;
+
+  return (
+    <>
+      <tr className={cssClasses.ROW}>
+        <Cell className={cssClasses.TITLE_CELL} colSpan={3}>
+          {/* Figure out how to not hard code colspans */}
```

Comment:
This cannot be done with colspans (as each table will have different numbers of columns) so I need to figure out another way to do this.

Reply:
Can we pass numColum in here? 😮